tests/unit/spread-shellcheck/must

```diff
@@ -0,0 +1,3 @@
+# files listed here must successfuly validate
```

Comment:
tests/unit/spread-shellcheck/must:1:25: "successfuly" is a misspelling of "successfully"

Reply:
Yeah, spellchecker is not enabled in `fundamental-mode` :unamused:

Reply:
Is that a `fundamental-bug` ;-)

Reply:
Fixed now.